ntc_templates/templates/hp_procurve_show_interfaces_brief.textfsm

```diff
@@ -20,11 +20,12 @@ showintbrief1
   ^\s+${PORT}\s+${TYPE}\s+\|\s+${INTRUSION_ALERT}\s+${ENABLED}\s+${STATUS}\s+${MODE}\s+(?:${MDI_MODE}\s+)?${FLOW_CTRL}\s*$$ -> Record
   ^\s*-+(?:\s|-|\+)+$$
   ^\s*$$
+  ^\s+\*.*$$
   ^. -> Error
 
 showintbrief2
   ^\s+${PORT}\s+${TYPE}\s+\|\s+${INTRUSION_ALERT}\s+${ENABLED}\s+${STATUS}\s+${MODE}\s+(?:${MDI_MODE}\s+)?${FLOW_CTRL}\s+${BCAST_LIMIT}\s*$$ -> Record
   ^\s*-+(?:\s|-|\+)+$$
   ^\s*$$
+  ^\s+\*.*$$
```

Comment:
Could this be something else other than `third-party`?, perhaps `^\s+\*\s+third\-party*$$` makes more sense.

Reply:
I'm assuming that if there is a * at the beginning of the line, it works like a postscript and can contain any data that should be ignored

Reply:
@itdependsnetworks for follow up.